lib/hipchat/room.rb

```diff
@@ -129,8 +129,8 @@ def send_message(message)
     # +notify+:: true or false
     #            (default false)
     def send(from, message, options_or_notify = {})
-      if from.length > 15
```

Comment:
Can you add a version check, and then, based on the version, check the lenght? It would be max 15 for v1, and max of 25 for v2.